### PR TITLE
Quality of Service queries

### PR DESCRIPTION
**Pull request by jackzampolin**

### CODE REVIEW

File: cmd/etl.go [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+	channeltypes "github.com/cosmos/ibc-go/modules/core/04-channel/types"
+	"github.com/cosmos/relayer/relayer"
+	_ "github.com/lib/pq"
+	"github.com/spf13/cobra"
+	"golang.org/x/sync/errgroup"
+)
+
+const driverName = "postgres"
+
+var db *sql.DB
```
Comment (jackzampolin) [inline cmd/etl.go:39]: These are just notes on the code, don't need to implement rn but when cleaning up:

Don't love the global here. Maybe create the `db *sql.DB` in the `qosCmd` and pass it into the sub functions?

File: cmd/etl.go [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+					block, err = chain.Client.Block(context.Background(), &h)
+					if err != nil {
+						return err
+					}
+
+					return nil
+				}, relayer.RtyAtt, relayer.RtyDel, relayer.RtyErr, retry.DelayType(retry.BackOffDelay), retry.OnRetry(func(n uint, err error) {
+					chain.LogRetryGetBlock(n, err, h)
+				})); err != nil {
+					if strings.Contains(err.Error(), "wrong ID: no ID") {
```
Comment (jackzampolin) [inline cmd/etl.go:145]: We could potentially lose blocks here for other error types maybe we need to log this?

File: cmd/etl.go [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+			}
+
+			for _, tx := range block.Block.Data.Txs {
+				sdkTx, err := chain.Encoding.TxConfig.TxDecoder()(tx)
+				if err != nil {
+					fmt.Printf("Failed to decode tx at height %d from %s \n", h, chain.ChainID)
+					return err
+				}
+
+				err = insertTxRow(tx.Hash(), chain.ChainID, h, block.Block.Time)
```
Comment (jackzampolin) [inline cmd/etl.go:158]: Do we potentially want to retry the insert in case of certain failures?

File: cmd/etl.go [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+
+		err := insertMsgTransferRow(hash, m.Token.Denom, m.SourceChannel, m.Route(), m.Token.Amount.String(), msgIndex)
+		if err != nil {
+			fmt.Printf("Failed to insert MsgTransfer. Index: %d Height: %d Err: %s", msgIndex, height, err.Error())
+		} else {
+			fmt.Printf("Wrote to database for height %d  \n", height)
+		}
+
+		fmt.Printf("[%s] => [%s]@{%d} *transfertypes.MsgTransfer [%x]\n", timestamp.String(), chainid, height, hash)
+		done()
```
Comment (jackzampolin) [inline cmd/etl.go:209]: IDK if this is needed anymore

File: cmd/etl.go [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+	for i := srcStart; i < srcCurrent; i++ {
+		srcBlocks = append(srcBlocks, i)
+	}
+	return srcBlocks, nil
+}
+
+func handleMsg(c *relayer.Chain, msg sdk.Msg, msgIndex int, height int64, timestamp time.Time, chainid string, hash []byte) {
+	switch m := msg.(type) {
+	case *transfertypes.MsgTransfer:
+		done := c.UseSDKContext()
```
Comment (jackzampolin) [inline cmd/etl.go:199]: i.e. in this code we don't interact with chains and we don't need to render addresses.

File: cmd/etl.go (hunk 2 above)
Comment (jtieri) [reply cmd/etl.go:146]: Added a logging statement here 

```
if strings.Contains(err.Error(), "wrong ID: no ID") {
    failedBlocks = append(failedBlocks, h)
} else {
    fmt.Printf("Failed to get block at height %d for chain %s. Err: %s \n", h, chain.ChainID, err.Error())
}
```

Off the top of my head I don't know what other errors would occur so for now I'll log them to stdout and if it looks like we hit errors that may be worthy of another retry we can handle them appropriately. 

Of course, if you know any errs we should be retrying here lmk and i'll add them!

**Pull request merged**